src/common/format.ts

```diff
@@ -0,0 +1,89 @@
+import { join as joinPath } from 'path';
+
+import { CompleteSuperfaceTestConfig } from '..';
+
+const ISO_DATE_REGEX =
+  /(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2}(?:\.\d*)?)((-(\d{2}):(\d{2})|Z)?)/gm;
+
+export const removeTimestamp = (payload: string): string =>
+  payload.replace(ISO_DATE_REGEX, '');
+
+export function getFixtureName(sfConfig: CompleteSuperfaceTestConfig): string {
+  const { profile, provider, useCase } = sfConfig;
+
+  return joinPath(
+    profile.configuration.id,
+    provider.configuration.name,
+    useCase.name
+  );
+}
+
+function testPayload(payload: string, match: string): boolean {
```

Comment:
Nitpick: I can't really tell what this function does form it's name. Many `isValidPayload` is better?